src/views/docs-view/consts.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/**
+ * So far, we have a pattern of using a common value for
+ * docs catchall route parameters: route/[[...page]].jsx.
+ * This default parameter ID captures that pattern.
+ * It can be overridden via options.
+ */
+export const DEFAULT_PARAM_ID = 'page'
+export const REMARK_ARRAY_ERROR =
```

Comment:
Moved this string into consts for two reasons:
- Remove strings from JavaScript files in favor of constants. This allows for better maintainability.
- Adjacent to the reason above: this string was used in both loaders